sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiverClient.cs

```diff
@@ -98,14 +92,6 @@ public abstract class ServiceBusReceiverClient : IAsyncDisposable
         ///
         internal TransportConsumer Consumer { get; }
 
-
-        /// <summary>
-        ///   The set of active Service Bus entity transport-specific consumers created by this client for use with
-        ///   delegated operations.
-        /// </summary>
-        ///
-        private ConcurrentDictionary<string, TransportConsumer> ActiveConsumers { get; } = new ConcurrentDictionary<string, TransportConsumer>();
```

Comment:
Could you please explain where were we using this before and why are we removing this?

Reply:
This looks like a remnant from Event Hubs where the consumer client was not bound to a transport and spawns a new transport consumer for each active operation, since they all required a different AMQP link/filter combination be opened.

I don't believe that is applicable in Service Bus, since the current design has the client acting in a way where it should be able to manage operations over a single active link.